code/modules/events/ion_storm.dm

```diff
@@ -566,6 +566,35 @@
 							message = "ALL [ionthreats] ARE NOW NAMED [ionobjects]."
 
 	return message
+	
+
+/proc/corp_name()

```

Comment:
This should probably go into wherever the names for the stock market corporations for cargo are